bahmni-lite/docker-compose.yml

```diff
@@ -397,6 +397,17 @@ services:
         - 'bahmni-queued-reports:/mounts/bahmni-queued-reports'
         - 'configuration_checksums:/mounts/configuration_checksums'
 
+  ipd:
+    image: bahmni/microfrontend-ipd:${IPD_TAG:?}
+    container_name: ipd
+    profiles: ["emr","bahmni-lite","bahmni-mart"]
+    restart: ${RESTART_POLICY}
+    environment:
+      TZ: ${TZ}
```

Comment:
@Arjun-Go I recall our earlier discussion regarding the IPD frontend repo and its reliance on a predefined timezone. As agreed, the timezone (TZ) environment variable can be removed, since the frontend shouldn't depend on a fixed timezone.

cc: @binduak, @mohan-13